src/lib.rs

```diff
@@ -462,30 +462,37 @@ fn median_select(data: &Vec<f64>, k: usize) -> Option<f64> {
     }
 }
 
-
 /*============================================ MAIN OPERATIONS : push, append, remove, pop ============================================= */
 impl SimpleAccumulator {
-    /// Similar to `push` in vector `Vec`. When `fixed_capacity` flag is 'true' and ring buffer is full
-    /// rewrites the oldest element with the latest, following FIFO order. 
+    /// Function similar to `push` in vector `Vec`. When `fixed_capacity` is 'true' and ring buffer is full,
```

Comment:
`default` should not set `fixed_capacity=true`.

Reply:
I don't think we should change how `default` trait works for `bool` types in rust, instead, we can make something like `SimpleAccumulator::FixedDefault` and that can set `fixed_capacity=false` for it.